src/v/cluster/topic_table.h

```diff
@@ -293,6 +355,16 @@ class topic_table {
     std::vector<std::invoke_result_t<Func, const topic_metadata_item&>>
     transform_topics(Func&&) const;
 
+    bool moving_from_node(
```

Comment:
nit: those two may be free static util functions

Reply:
fixed